### PR TITLE
Implement flexible DOT payments

### PR DESCRIPTION
**Pull request by semuelle**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ title: 🙋 FAQ
...
+  Every milestone payment will have two parts: one in DOT, vesting linearly over two years. The percentage of your grant paid in DOT ranges from 50 to 100% and will be negotiated during the application phase. For this, we need a Polkadot address in the grant agreement. It has to be a native Polkadot address as this is the only way to receive vested DOT that can be staked. The remaining percentage of the payment is in the currency of your choice (see the application template for options). For the payment, please submit two invoices: one for the currency part (if applicable), one for the DOT part. Since the grant agreements are denominated in USD, we ask for invoices denominated in USD; we will convert the agreed upon percentage to DOT at the day of payment based on the previous day's closing price on [CoinMarketCap](https://coinmarketcap.com). Our [invoice form](https://docs.google.com/forms/d/18BdmYEt8gvcC-5f7TMHdJftBr9dEeA2_lUfIqkH7it8/viewform) will walk you through the process.
```
Comment (takahser) [inline docs/faq.md:20]: > The remaining percentage of the payment is in the currency of your choice (see the application template for options).

Four points:

1. I suggest to say "FIAT" or "FIAT currency" to make it clear
2. We would also support stables here, right? That's currently not clear
3. What are you referring on the application template?
4. Do we really support any fiat currency "of your choice"? We might get some exotic requests, if we do that.

**Review by takahser — CHANGES_REQUESTED** [review]
@semuelle thanks & good job. I just fixed 1 typo and added some questions regarding the FIAT part.

File: docs/Support Docs/T&Cs.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ The Parties agree that the Grant is a lump-sum payment and that no additional co
...
 
-The Grantee accepts and acknowledges that the Grant is a combination of a lump-sum cash payment (the “Cash Grant”) and a DOT component (the “Locked DOT Grant”), together hereinafter referred to as the "Grant", and that no additional compensation is due for the actual development costs incurred. The Grant is structured as milestone payments for the accomplishment of work during several phases of the Services Delivery as indicated in Annex 1 (the "Milestones").
+The Grantee accepts and acknowledges that the Grant is a combination of a lump-sum cash payment (the “Cash Grant”) and a DOT component (the “Locked DOT Grant”), together hereinafter referred to as the "Grant", and that no additional compensation is due for the actual development costs incurred. We may decide the proportion of the Locked DOT Grant at our sole and absolute discretion, and the Locked DOT Grant may amount to up to 100% of the overall Grant amount. The Grant is structured as milestone payments for the accomplishment of work during several phases of the Services Delivery as indicated in Annex 1 (the "Milestones").
```
[the remark this answers is not in the document]
Comment (semuelle) [reply docs/Support Docs/T&Cs.md:170]: "overal"?

File: docs/faq.md (hunk 1 above)
Comment (semuelle) [reply docs/faq.md]: I have made it a bit clearer, but the text specifically mentions "see the application template for options". There, it lists USDC, USDT and fiat. Hence the somewhat vague term "currency".

File: docs/Support Docs/T&Cs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (takahser) [reply docs/Support Docs/T&Cs.md:170]: not sure what got into me when I wrote this 😂

**Review by takahser — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Pull request merged**